com.unity.render-pipelines.high-definition/Runtime/Debug/DebugLightVolumes.shader

```diff
@@ -68,6 +69,7 @@ Shader "Hidden/HDRP/DebugLightVolumes"
             HLSLPROGRAM
             #pragma vertex vert
             #pragma fragment frag
+            #pragma only_renderers d3d11 playstation xboxone xboxseries vulkan metal switch
```

Comment:
Can you do a separate PR for this one as it will need to be backport, thanks

Reply:
(and same for other like this)

Reply:
https://github.com/Unity-Technologies/Graphics/pull/6204